tests/Http/Legacy/Two/ActivityApiTest.php

```diff
@@ -307,25 +304,25 @@ public function testQuantityOnActivityIndex()
     }
 
     /**
-     * Test for retrieving a user's activity with northstar_id and campaign_run_id query params.
+     * Test for retrieving a user's activity with northstar_id and campaign_id query params.
      *
      * GET /activity?filter[]=
      * @return void
      */
     public function testActivityIndexWithNorthstarIdAndCampaignRunIdFilters()
     {
-        $signup = factory(Signup::class)->create(['northstar_id' => 17, 'campaign_run_id' => 143]);
```

Comment:
I think we should change the title of this test to `testActivityIndexWithNorthstarIdAndCampaignIdFilters`

Reply:
Ah, of course! Updated in 4227bd7.